lib/netsuite/actions/add.rb

```diff
@@ -77,7 +77,7 @@ def add(credentials={})
           @errors = response.errors
 
           if response.success?
-            @internal_id = response.body[:@internal_id]
+            @internal_id = response.body[:@internal_id] if response.body.respond_to? :[]
```

Comment:
@ahuth Yeah, in the case of uploading media we don't get a response body back, just a response of true.